README.md

```diff
@@ -22,7 +22,7 @@ Nethermind documentation is available at [docs.nethermind.io](https://docs.nethe
 
 ### Supported networks
 
-**`Mainnet`** **`Sepolia`** **`Holesky`** **`Gnosis`** **`Chiado`** **`OP Mainnet`** **`OP Sepolia`** **`Base Mainnet`** **`Base Sepolia`** **`Energy Web`** **`Volta`**
+**Ethereum** · **Gnosis** · **Optimism** · **Base** · **Taiko** · **Energy Web**
```

Comment:
maybe it would be good to mention somewhere that it also includes testnets and list them?

Reply:
We have all those networks in our docs linked above.